ast/ast.go

```diff
@@ -316,6 +316,13 @@ func (*RenameProc) ProcNode()     {}
 func (*FuseProc) ProcNode()       {}
 func (*JoinProc) ProcNode()       {}
 
+// A FunctionCall can also represent a proc that is either a group-by
```

Comment:
```suggestion
// A FunctionCall represents a proc that is either a group-by
```

Reply:
That comment was explaining why it was a method on procNode but I moved it to the site of the type definition and clarified the comment.